CONTRIBUTING.md

```diff
@@ -121,21 +120,23 @@ bin/swagger_generate
 
 ## Dependency management
 
+Dependencies are managed by go modules. Make sure you have at least go1.11 installed.
+
 * Install project's frozen packages
 ```bash
-dep ensure
+go mod download
```

Comment:
Do we really need to make this step explicit? All go commands respect modules and will download any missing deps.